opencti-platform/opencti-graphql/src/database/redis.ts

```diff
@@ -90,14 +90,26 @@ const clusterOptions = (): ClusterOptions => ({
 
 export const createRedisClient = (provider: string, autoReconnect = false): Cluster | Redis => {
   let client: Cluster | Redis;
-  const isCluster = conf.get('redis:mode') === 'cluster';
-  if (isCluster) {
+  const redisMode: string = conf.get('redis:mode');
+
+  if (redisMode === 'cluster') {
     const clusterNodes = generateClusterNodes(conf.get('redis:hostnames') ?? []);
     client = new Redis.Cluster(clusterNodes, clusterOptions());
+  } else if (redisMode === 'sentinel') {
+    const sentinelConfiguration: RedisOptions = {
+      sentinels: [
+        { host: conf.get('redis:sentinel_leader_host'), port: conf.get('redis:sentinel_leader_port') },
+        { host: conf.get('redis:sentinel_additional_host'), port: conf.get('redis:sentinel_additional_port') },
```

Comment:
Not possible to have a list instead of limiting to 2 entries?

Reply:
Yep could be, don't know how that didn't hit me before pushing... Can even reuse the generateClusterNodes() func.

Has some impact on internal platform definitions, will update the PR when everything is aligned